src/components/NewTodo/NewTodo.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import classNames from 'classnames';
+
+class NewTodo extends React.Component {
+  state = {
+    selectedUser: 0,
+    inputTodoValue: '',
+    inputTodoError: false,
+    selectUserError: false,
+  }
+
+  onChangeSelectOfUser = ({ target: { value } }) => {
+    if (value !== 0) {
```

Comment:
```suggestion
    if (value) {
```
`0` в булевом значении будет `false`